lib/app.js

```diff
@@ -1,53 +1,44 @@
 (function (factory) {
   if (typeof define === 'function' && define['amd']) {
-    define(['knockout', './calculator', './todo-list'], factory);
+    define(['knockout'], factory);
   } else if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
-    factory(require('knockout'), require('./calculator'), require('./todo-list'));
+    factory(require('knockout'));
   } else {
-    factory(ko, Calculator, ToDoList);
+    factory(ko);
   }
-} (function (ko, Calculator, ToDoList) {
+} (function (ko) {
 
   function App() {
-    let calculators = ko.observableArray();
-
+    let widgets = ko.observableArray();
 
     function addCalculator() {
-      calculators.push({});
-    }
-
-    function removeCalculator() {
-      calculators.remove(this);
+      widgets.push({ name: 'calculator' });
     }
 
-    let toDoLists = ko.observableArray();
-
     function addToDoList() {
-      toDoLists.push({});
+      widgets.push({ name: 'todo-list' });
     }
 
-    function removeToDoList() {
-      toDoLists.remove(this);
+    function remove() {
+      widgets.remove(this);
```

Comment:
Because we are using objects, we can simply do `widgets.remove(this)` as each object will be unique